Source/WebCore/css/parser/CSSPropertyParserConsumer+Animations.h

```diff
@@ -35,6 +35,8 @@ struct CSSParserContext;
 
 namespace CSSPropertyParserHelpers {
 
+Vector<std::pair<CSSValueID, double>> consumeKeyframeKeyList(CSSParserTokenRange&);
```

Comment:
Assuming this is meant to be parsing the updated production of `<keyframe-selector>` defined https://drafts.csswg.org/scroll-animations-1/#named-range-keyframes, would you mind adding production name and URL as done below?